src/tasks/installers/operator.ts

```diff
@@ -120,6 +125,11 @@ export class OperatorTasks {
       },
       {
         title: `Create ClusterRoleBinding ${this.operatorClusterRoleBinding}`,
+        skip: () => {
+          if (!flags['os-oauth']) {
+            return `No need for ClusterRole ${this.operatorClusterRole}`
```

Comment:
Previously it was implemented in such way, but then I removed such check during console link support implementing.

How consolelink supposed to work after your changes? https://github.com/eclipse/che-operator/blob/master/deploy/cluster_role.yaml#L41

Reply:
https://github.com/che-incubator/chectl/pull/291

Reply:
I think the solution should be smarter - chectl can check is the current user has admin rights:
- if has - create cluster role, everything is OK
- if it does not - OAuth flag is not permitted, during deploying chectl prints warning that consolelink won't be created.

It's needed to check what che-operator will do on OpenShift 4.2 when he does not permissions to create console link, probably it fails to deploy Che...